extensions/amp-story/1.0/amp-story.js

```diff
@@ -1494,12 +1498,26 @@ export class AmpStory extends AMP.BaseElement {
     if (height === 0 && width === 0) {
       return;
     }
-    this.storeService_.dispatch(Action.SET_PAGE_SIZE, {height, width});
     setImportantStyles(this.win.document.documentElement, {
       '--story-dvh': px(height / 100),
     });
   }
 
+  /**
+   * Handles resize events and sets the store service's width and height.
+   * @private
+   */
+  onViewportResize_() {
+    const page = this.element.querySelector(`amp-story-page[active]`);
+
+    const pageSize = {
```

Comment:
I'd check to see how this compiles down, but you likely do not need to call `getLayoutBox` twice.

```
const layoutBox = page?./*OK*/ getLayoutBox();
const pageSize = {'width':layoutBox?.width, 'height':layoutBox?.height};
this.storeService_.dispatch(...);
```

This should compile down to...
```
let c=b?.getLayoutBox();
this.storeService_.dispatch(..., {'width':c?.width, 'height':c?.height});
```

But if you're planning to refactor, I'd consider just storing the `layoutBox` in the dispatch, since it's a reduction of the `layoutBox` object.